website/docs/5.0-upgrade-guide.html.markdown

```diff
@@ -57,6 +57,10 @@ Please follow the format in the example below for listing breaking changes in re
 * The `example_property_with_changed_default` property now defaults to `NewDefault`.
 ```
 
+### `azurerm_security_center_auto_provisioning`
+
+* This deprecated resources has been removed from the Azure Provider.
+
```

Comment:
This should be under the `### Removed Resource` section and it would be good to provide a link to the deprecation message as well
```suggestion
### `azurerm_security_center_auto_provisioning`

* This deprecated resource has been removed from the Azure Provider.

```